tests/version/test_markers.py

```diff
@@ -1562,25 +1567,20 @@ def test_only(marker: str, only: list[str], expected: str) -> None:
                 ' or python_version >= "3.9"'
             ),
             "~3.7 || ~3.9",
-            'sys_platform == "linux"',
+            "",
         ),
         (
             (
                 'python_version < "3.8" or sys_platform == "linux"'
                 ' or python_version >= "3.9" or sys_platform == "win32"'
             ),
             "~3.7 || ~3.9",
-            'sys_platform == "linux" or sys_platform == "win32"',
```

Comment:
I removed these test because with the fix they are not that intesting anymore because they are too similar to the test above.